testapps/test_pillowtop/tests/test_form_pillow.py

```diff
@@ -22,15 +30,23 @@ def setUp(self):
         super(FormPillowTest, self).setUp()
         FormProcessorTestUtils.delete_all_xforms()
         self.pillow = get_xform_pillow(skip_ucr=True)
-
         factory = AppFactory(domain=self.domain)
         self.app = factory.app
         self.app.save()
 
+    @classmethod
+    def setUpClass(cls):
+        cls.es = get_es_new()
+        initialize_index_and_mapping(cls.es, USER_INDEX_INFO)
+
     def tearDown(self):
         self.app.delete()
         super(FormPillowTest, self).tearDown()
 
+    @classmethod
+    def tearDownClass(cls):
+        ensure_index_deleted(USER_INDEX)
```

Comment:
tearDownClass must call super
```suggestion
        ensure_index_deleted(USER_INDEX)
        super().tearDownClass()
```